### PR TITLE
Add statements mappings to snforge test artifacts

### PR DESCRIPTION
**Pull request by piotmag769**

### CODE REVIEW

**Review by MaksymilianDemitraszek — APPROVED** [review]
please address the requested changes

File: extensions/scarb-snforge-test-collector/src/main.rs [hunk 1]
```diff
@@ -22,6 +22,13 @@ mod metadata;
 struct Args {
     #[command(flatten)]
     packages_filter: PackagesFilter,
+
+    /// Unstable flag used to add a mapping between sierra statement indexes and fully qualified
+    /// paths of cairo functions to debug info.
+    /// A statement index maps to a function which cause the statement to be generated.
+    /// Used by [cairo-profiler](https://github.com/software-mansion/cairo-profiler).
+    #[arg(long)]
+    generate_statements_functions_mappings: bool,
```
[the remark this answers is not in the document]
Comment (piotmag769) [reply extensions/scarb-snforge-test-collector/src/main.rs:31]: forgor

**Comment by piotmag769** [comment]
@maciektr can we merge it?

**Comment by piotmag769** [comment]
@maciektr done

**Review by maciektr — APPROVED** [review]

**Pull request merged**